### PR TITLE
Display section in gradebook

### PR DESCRIPTION
**Pull request by kevincharles**

### CODE REVIEW

File: src/Tools/_framework/ToolPanels/GradebookStudent.jsx [hunk 1]
```diff
@@ -267,12 +267,16 @@ export default function GradebookStudent() {
...
 
   let studentName = `${students.contents[userId]?.firstName} ${students.contents[userId]?.lastName}`;
+  const section = students.contents[userId]?.section;
 
   return (
     <>
       <div style={{ marginLeft: "18px" }}>
         <b>Gradebook for {studentName}</b>
       </div>
+      <div style={{ marginLeft: "18px" }}>
```
Comment (dqnykamp) [inline src/Tools/_framework/ToolPanels/GradebookStudent.jsx:277]: If section is blank, then eliminate this line.

File: src/Tools/_framework/ToolPanels/GradebookStudentAssignment.jsx [hunk 2]
```diff
@@ -389,12 +388,15 @@ export default function GradebookStudentAssignmentView() {
...
 
   let studentName = `${students.contents[userId]?.firstName} ${students.contents[userId]?.lastName}`;
-
+  const section = students.contents[userId]?.section;
   return (
     <>
       <div style={{ marginLeft: "18px" }}>
         <b>Gradebook for {studentName}</b>
       </div>
+      <div style={{ marginLeft: "18px" }}>
```
Comment (dqnykamp) [inline src/Tools/_framework/ToolPanels/GradebookStudentAssignment.jsx:397]: If section is blank, then eliminate this line.

**Review by dqnykamp — CHANGES_REQUESTED** [review]
In the two student views, eliminate the "Section " line if section is blank.

File: src/Tools/_framework/ToolPanels/GradebookStudent.jsx (hunk 1 above)
Comment (kevincharles) [reply src/Tools/_framework/ToolPanels/GradebookStudent.jsx]: Handled section

**Review by dqnykamp — APPROVED** [review]
+1

**Pull request merged**